docs/docs/development/local-toolchain/setup/docker/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Docker
+sidebar_label: Docker
+---
+
+### Source Code
+
+First, you'll need to clone the ZMK source repository if you haven't already.
+Open a terminal and navigate to the folder you would like to place your `zmk`
+directory in, then run the following command:
+
+```sh
+git clone https://github.com/zmkfirmware/zmk.git
+```
+
+### Docker Setup
+
+Currently, there are two ways of building your local development environment:
+
+- [VS Code](vscode.md): \
+  This approach takes advantage of [VS Code](https://code.visualstudio.com/)’s
+  [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+  extension, allowing you to open the ZMK repository in a Docker container
+  directly within the editor. It is recommended for everyone who is familiar
+  with or already uses [VS Code](https://code.visualstudio.com/), since it
+  easily integrates with Docker and provides a seamless development experience.
+
+- [Dev Container CLI](cli.md): \
+  For users who prefer not to use [VS Code](https://code.visualstudio.com/), the
+  [Dev Container CLI](https://github.com/devcontainers/cli) provides a
+  command-line alternative for managing and running
+  [dev containers](https://containers.dev/), with similar functionality.
```

Comment:
I would prefer if this was done through the use of tabs, rather than separate pages. Alternatively, we could consider adding the cli section at the same depth as the native section, but I'm not a fan of having this nested so deep.

Reply:
As suggested I merged everything together by using tabs. Now there is only the Docker page. Hope this is what you had in mind.